### ISSUE TITLE
Error when running `abctl local install` on Mac arm

### ISSUE THREAD

**Issue by e-larsson** [airbytehq/airbyte#40745]
### What happened?

Following the [install guide](https://docs.airbyte.com/using-airbyte/getting-started/oss-quickstart) to install Airbyte, but I run into an error when I run `abctl local install`.

- I have tried to install `abctl` via the three method provided.
- I have tried the latest GO version `1.22.5` and the previous `1.21.12`.
- I have tried with native arm Docker and with Rosetta 2.

Same error regardless. Not sure what to do 🤷 

```
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/airbytehq/abctl/internal/telemetry.loadConfigFromFile({0xc00019a120, 0x29})
	github.com/airbytehq/abctl/internal/telemetry/config.go:140 +0x534
github.com/airbytehq/abctl/internal/telemetry.Get.func1({0xa8?, {0xc000056065?, 0xc0005f3ae8?}, 0x0?})
	github.com/airbytehq/abctl/internal/telemetry/client.go:104 +0xc7
github.com/airbytehq/abctl/internal/telemetry.Get({0x0, 0x0, 0x3?})
	github.com/airbytehq/abctl/internal/telemetry/client.go:127 +0x237
github.com/airbytehq/abctl/internal/cmd/local.NewCmdLocal.func1(0xc00061b1a8?, {0x54d365c?, 0x4?, 0x54d3640?})
	github.com/airbytehq/abctl/internal/cmd/local/local.go:28 +0xbf
github.com/spf13/cobra.(*Command).execute(0xc000495208, {0x6eccbe0, 0x0, 0x0})
	github.com/spf13/cobra@v1.8.0/command.go:954 +0x951
github.com/spf13/cobra.(*Command).ExecuteC(0xc000494908)
	github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	github.com/spf13/cobra@v1.8.0/command.go:1032
github.com/airbytehq/abctl/internal/cmd.Execute({0x5d74ba0?, 0xc00033eb90?}, 0x77359400?)
	github.com/airbytehq/abctl/internal/cmd/cmd.go:41 +0x4f
main.main()
	github.com/airbytehq/abctl/main.go:45 +0x1a7
```

### What did you expect to happen?

No error message

### Abctl Version

<details>

```console
$ abctl version
version: v0.6.0
```

</details>

### Docker Version

<details>

```console
$ docker version
Client:
 Version:           26.1.4
 API version:       1.45
 Go version:        go1.21.11
 Git commit:        5650f9b
 Built:             Wed Jun  5 11:26:02 2024
 OS/Arch:           darwin/arm64
 Context:           desktop-linux

Server: Docker Desktop 4.31.0 (153195)
 Engine:
  Version:          26.1.4
  API version:      1.45 (minimum version 1.24)
  Go version:       go1.21.11
  Git commit:       de5c9cf
  Built:            Wed Jun  5 11:29:12 2024
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.6.33
  GitCommit:        d2d58213f83a351ca8f528a95fbd145f5654e957
 runc:
  Version:          1.1.12
  GitCommit:        v1.1.12-0-g51d5e94
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

</details>

### OS Version

<details>

```console
$ uname -a
Darwin Tokaimura.lan 23.5.0 Darwin Kernel Version 23.5.0: Wed May  1 20:14:38 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T6020 arm64
```

</details>

**Comment by marcosmarxm** [comment]
@e-larsson can you share the error message?

**Comment by e-larsson** [comment]
> @e-larsson can you share the error message?

Oops sorry 🤦  Updated the Issue

**Comment by marcosmarxm** [comment]
@colesnodgrass do you mind helping here?

**Comment by Christoffersn** [comment]
I get the same error.
If it helps - the issue seems to have been introduced in version 0.5.0 of abctl.
abctl v0.4.0 runs on my Macbook Pro M2, but v0.5.0 does not, and instead fails with the same error:
`panic: interface conversion: interface {} is nil, not string`

**Comment by colesnodgrass** [comment]
Thanks for reporting this issue.  This has been fixed in the [v0.7.2](https://github.com/airbytehq/abctl/releases/tag/v0.7.2) release.
